src.md

```
@@ -1,209 +1,42 @@
-  - Response: 404 Not Found repeated multiple times.
-- Server logs:
-  - `GET /api/keys/initial-session/<conversationId>/<sessionId> 404 ...`
-  - `No online users found in convo <conversationId> to fulfill key request`
-  - `POST /api/prekey-bundle 201` — prekey bundle upload succeeded.
-  - `POST /api/session-keys/<conversationId>/ratchet 201` — ratchet request succeeded (server generated session keys).
-- Code mapping:
-  - `client decryptMessage` calls `GET /api/keys/initial-session` (see crypto.ts) when local key is absent.
-  - Server `GET /api/keys/initial-session` returns 404 if either `keyRecord` is missing or `keyRecord.initiatorEphemeralKey` is null or if initiator's `User.publicKey` is null (see keys.ts).
-  - `rotateAndDistributeSessionKeys` (server ratchet in sessionKeys.ts) currently doesn't include `initiatorEphemeralKey` in `createMany` data; conversely, conversations.ts uses `initialSession` payload and includes `initiatorEphemeralKey` when client-provided.
+Format Pesan (Markdown): Mengizinkan pengguna untuk memformat teks menggunakan
+      Markdown sederhana, seperti *tebal*, _miring_, ~coret~, atau  ``blok kode`` . Fitur
```

markdownlint-cli2 (0.18.1)

Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)